forge-gui/res/cardsfolder/upcoming/draconic_debut.txt

```diff
@@ -1,11 +1,13 @@
 Name:Draconic Debut
 ManaCost:X R
 Types:Sorcery
-A:SP$ DealDamage | ValidTgts$ Any | NumDmg$ X | TargetMin$ 0 | SubAbility$ DBEffect | SpellDescription$ CARDNAME deals X damage to any target.The next Dragon creature spell you cast costs {X} less to cast.
-SVar:DBEffect:DB$ Effect | StaticAbilities$ ReduceCost | Triggers$ TrigCastSpell | Duration$ Permanent
-SVar:ReduceCost:Mode$ ReduceCost | EffectZone$ Command | Type$ Spell | Activator$ You | ValidCard$ Creature.Dragon | Amount$ X | Description$ The next Dragon creature spell you cast costs {X} less to cast.
-SVar:TrigCastSpell:Mode$ SpellCast | ValidActivatingPlayer$ You | TriggerZones$ Command | Execute$ RemoveEffect | Static$ True
+A:SP$ DealDamage | ValidTgts$ Any | NumDmg$ X | TargetMin$ 0 | SubAbility$ DBStore | SpellDescription$ CARDNAME deals X damage to any target. 
+SVar:DBStore:DB$ StoreSVar | SVar$ Num | Type$ CountSVar | Expression$ X | SubAbility$ DBEffect
```

Comment:
@tool4ever isn't there a cleaner way to get X amount to the Effect without having to use StoreSVar?

I see the StoreChosenNumber - that might be one tactic

Reply:
Any examples I should look at ?

Reply:
#4024

Reply:
Thanks